release-notes/unreleased/2520-Show-number-of-search-results.yml

```
@@ -0,0 +1,7 @@
+issue_key: 2520
+show_in_stores: true
+platforms:
+  - web
+  - android and ios
```

:x: This won't work, this will be parsed by some of our scripts which require valid yaml files:
```suggestion
  - android
  - ios
```

For more information on yaml, have a look at https://www.yaml.info/learn/index.html or some other site :)